recipes/fastscapelib/meta.yaml

```diff
@@ -0,0 +1,76 @@
+{% set name = "fastscapelib" %}
+{% set version = "0.1.2" %}
+{% set sha256 = "d576805c1ff282af2204bd999ad0d6757c2748df13ca6a21914bd484d5d5db59" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/fastscape-lem/fastscapelib/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win and vc<14]
+
+outputs:
+  - name: fastscapelib
+    script: install_cpp_headers.sh   # [unix]
+    script: install_cpp_headers.bat  # [win]
+    build:
+      activate_in_script: True
+    requirements:
+      build:
+        - {{ compiler('cxx') }}
+        - cmake
+      host:
+        - xtensor >=0.17.1,<0.18
+      run:
+        - xtensor >=0.17.1,<0.18
+    test:
+      commands:
+        - test -f ${PREFIX}/include/fastscapelib/fastscapelib.hpp  # [unix]
+        - test -f ${PREFIX}/lib/cmake/fastscapelib/fastscapelibConfig.cmake  # [unix]
+        - test -f ${PREFIX}/lib/cmake/fastscapelib/fastscapelibConfigVersion.cmake  # [unix]
+        - if exist %LIBRARY_PREFIX%\include\fastscapelib\fastscapelib.hpp (exit 0) else (exit 1)  # [win]
+        - if exist %LIBRARY_PREFIX%\lib\cmake\fastscapelibConfig.cmake (exit 0) else (exit 1)  # [win]
+        - if exist %LIBRARY_PREFIX%\lib\cmake\fastscapelibConfigVersion.cmake (exit 0) else (exit 1)  # [win]
+  - name: fastscapelib-python
+    script: install_python.sh   # [unix]
+    script: install_python.bat  # [win]
+    build:
+      activate_in_script: True
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        - cmake
+      host:
+        - python
+        - setuptools
+        - numpy
+        - xtensor >=0.17.1,<0.18
+        - xtensor-python >=0.20,<0.21
+      run:
```

Comment:
Does this not depend on fastscapelib?

Reply:
No, Python `setup.py` calls cmake, which directly uses fastscapelib C++ headers in the same repository.

Actually, I've chosen to reuse the same cmake base script for both the C++ library and the Python library because there is no reason to maintain independent versions (that's also why I added one single recipe for the two packages).